compatibility/history-proxy/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "gui_proxy",
+  "version": "0.0.1",
+  "main": "index.js",
+  "author": {
+    "name": "Francisco Angelo Cabelo",
+    "email": "francisco.a.cabelo@gmail.com",
+    "url": "https://github.com/cfrancisco"
+  },
+  "description": "The GUI Proxy allows the Dojot GUI v1 to be compliant with InfluxDb, which is used in newer versions of Dojot.",
+  "scripts": {
+    "start": "node ./src/index.js",
+    "test": "jest --verbose",
+    "lint": "eslint --ext .js .",
+    "coverage": "jest --ci --coverage && codecov",
+    "debug": "node --trace-sync-io  ./src/index.js",
+    "check-licenses": "apache2-license-checker"
+  },
+  "jest": {
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true,
+    "testEnvironment": "node",
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
+    ]
+  },
+  "license": "Apache-2.0",
+  "licenses": [
+    {
+      "type": "Apache-2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0"
+    }
+  ],
+  "keywords": [
+    "proxy",
+    "dojot",
+    "gui"
+  ],
+  "private": false,
+  "devDependencies": {
+    "@types/jest": "^26.0.10",
+    "apache2-license-checker": "^1.0.0",
+    "eslint": "^7.19.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.21.1",
+    "jest": "^26.6.3",
+    "prettier": "^2.1.1"
+  },
+  "dependencies": {
+    "@dojot/microservice-sdk": "0.1.9",
+    "axios": "^0.21.1",
+    "codecov": "^3.8.1",
```

Comment:
codecov it's not necessary..
try to follow the pattern of other services in this repo